receiver/mysqlreceiver/metadata.yaml

```diff
@@ -1,5 +1,10 @@
 name: mysqlreceiver
 
+resource_attributes:
+  endpoint:
```

Comment:
Resource attributes are part of a single hierarchy that must cover the whole project, so we need to name this more specifically.

I suggest `mysql.instance.endpoint`. In the future, when we add mysql database specific metrics, we'll likely add `mysql.database.name` and/or `mysql.database.id`, and those metrics will belong to a resource with both instance and database attributes.